setup.cfg

```diff
@@ -1,2 +1,7 @@
 [bdist_wheel]
 universal=1
+
+[check-manifest]
```

Comment:
Does this mean we exclude the Makefile and tests from the sdist? I think it's common practice to include them but I could be wrong.

Reply:
Hmm actually I think is more common to not include the tests that to include them (but I may be wrong 🤷)

Reply:
@nixjdm ?

Reply:
It is default behaviour to include them at least (https://docs.python.org/3/distutils/sourcedist.html), and at least Flask includes them. Though that doesn't mean it's good practice.

My understanding is that you should be able use an sdist as a snapshot of all the things needed for development, this includes docs, Makefiles and tests.

Reply:
Ok, will update:-)

Reply:
Fixed!